extension/js/common/platform/store/contact-store.ts

```diff
@@ -14,14 +13,36 @@ import { OpenPGPKey } from '../../core/crypto/pgp/openpgp-key.js';
 type DbContactObjArg = {
   email: string,
   name?: string | null,
-  client?: 'pgp' | 'cryptup' | PgpClient | null,
   pubkey?: string | null,
   pendingLookup?: boolean | number | null,
   lastUse?: number | null, // when was this contact last used to send an email
-  lastSig?: number | null, // last pubkey signature (when was pubkey last updated by owner)
   lastCheck?: number | null; // when was the local copy of the pubkey last updated (or checked against Attester)
 };
 
+type Email = {
+  email: string;
+  name: string | null;
+  searchable: string[];
+  fingerprints: string[];
+  pendingLookup: number;
+  lastUse: number | null;
+};
+
+type Pubkey = {
+  fingerprint: string;
+  armoredKey: string;
+  longids: string[];
+  lastCheck: number | null,
+  lastSig: number;
```

Comment:
@tomholub  Turns out we don't need to store this field, as we don't index it, nor search by it and never extract its value (the key is always parsed and `lastModified` of the parsed key is returned instead)

Reply:
sounds good, can drop